plugins/available/vagrant-as-root.plugin.bash

```diff
@@ -0,0 +1,4 @@
+# Simple function for vagrant ssh that immediately switches to root.
+function vsshr() {
+      vagrant ssh $1 -- -t 'sudo su -; /bin/bash'
+  }
```

Comment:
There's an extra space before the closing `}`, please remove that.

And please add an empty line after the last line of code, if it's not already there (can't see this in the diff).